testsuite/MDAnalysisTests/analysis/test_distances.py

```diff
@@ -235,3 +235,136 @@ def test_between_return_type(self, dists, group, ag, ag2):
             dists
         )
         assert isinstance(actual, MDAnalysis.core.groups.AtomGroup)
+
+
+class TestAtomicDistances(object):
+    @staticmethod
+    @pytest.fixture()
+    def ad_u():
+        return MDAnalysis.Universe(GRO, XTC)
```

Comment:
can we use a `mda.Universe.empty()` here to avoid having to read files for a test?

For its positions, you could either use a seeded random number, or just something like `np.arange(natoms * 3).reshape(natoms, -1)` to generate arbitrary coordinates

Reply:
So in order to do this, I had to change the `dist()` tests to use `calc_bonds()` instead because the Universe does not have resids